package.json

```diff
@@ -31,7 +31,7 @@
     "execa": "5.1.1",
     "fs-extra": "10.1.0",
     "js-yaml": "4.1.0",
-    "minimatch": "5.1.0",
+    "micromatch": "^4.0.5",
```

Comment:
What's the difference between those two? In my research I just found minimatch. Didn't know there was an alternative.

Reply:
More features and it performs better than minimatch